### ISSUE TITLE
Same Flashcards Showing Up Twice

### ISSUE THREAD

**Issue by pormonto** [remnoteio/remnote-issues#606]
Some flashcards repeat themselves although I had already gone through them. For example, on the Android app, I had the last card repeat itself twice and acting like the spaced repetition period didn't exist. The first time it showed it gave me the options to repeat the same material in a couple days, and then in the repeated card, it gave the choice to postpone the card up to month. I just made this card a couple days ago.

More problems like this happen where I finish my queue in the morning (I did it with my browser today) and when switch to my phone, it gives me part that same queue again even though I had finished it earlier that day. Note that it didn't give me the entire queue, just part of it.

Extra details (may be jumbled up an incoherent but it might help): When entering the app, it was trying recognize that did some if earlier, so it turned the 27 to 0. A little later I see that queue shows up again and has stars next to it. I press it and it the same queue I did earlier that day (still referring to the one only showed part of it).

**To Reproduce**
I'm not exactly sure what triggered this, but try to switch between the Android app and the Web version to see if any inconsistencies happen.

**Expected behavior**
I expected for the material to be shown once and not multiple times in the same day.
I remember when looking at the tutorial videos and hearing something that bothered me: when you do flashcards without spaced repetition, spaced repetition still applies. That's crazy! I imagined there would be a checker in the code to see if the card has been reviewed within a certain time interval (aka the spaced repetition time). What also confuses me is how the cards in that mode can't be put into a state where spaced repetition does not affect them. At the least, you could make it so each option temporarily works like this option 
![image](https://user-images.githubusercontent.com/72433858/105554571-76b34800-5cbc-11eb-8d62-34ac247f259b.png). You could even temporarily remove the options altogether. Sorry for the rant, but I believe that detail has something to do with why the card didn't recognize it was being used twice. Another option would be just keep the cards from changing spaced repetition dates if they were reviewed in the same day, but that wouldn't fix the underlying problem in my opinion. Alright, that's everything to the detail.

![image](https://user-images.githubusercontent.com/72433858/105554868-12dd4f00-5cbd-11eb-95d7-1327513d55c4.png)
![image](https://user-images.githubusercontent.com/72433858/105554893-1f61a780-5cbd-11eb-93d2-783632029072.png)

I also have a completely different thing happening on my phone. Same thing as the first thing I said.
![image](https://user-images.githubusercontent.com/72433858/105555529-6a2fef00-5cbe-11eb-9045-e0e6cfbf6462.png)
![image](https://user-images.githubusercontent.com/72433858/105556185-107bf480-5cbf-11eb-866d-353a612fe58c.png)

**Desktop:**
 Chrome 84.0.4147 / Chrome OS 13099.110.0

**Smartphone:**
 - Device: [Moto e6]
 - OS: [Android 9]
 - Android App

**Additional context**
TLDR: My phone and computer are showing to different queues at the same time. Also they're glitching so they get counted twice.

**Comment by pormonto** [comment]
Update: I did the queue on the phone and it showed me the days option again
![image](https://user-images.githubusercontent.com/72433858/105556748-7c5e5d00-5cbf-11eb-8d72-507fe90835ad.png)


This caused the web version to update the queue from 10 to 1 now.
![image](https://user-images.githubusercontent.com/72433858/105556726-736d8b80-5cbf-11eb-9020-ee7ce652f131.png)


I already did this one but I'm glad it's giving me the days option again.
![image](https://user-images.githubusercontent.com/72433858/105556822-a6178400-5cbf-11eb-9795-c0c972255838.png)


It's fixed??
![image](https://user-images.githubusercontent.com/72433858/105556893-d6f7b900-5cbf-11eb-832e-c28dc15e2859.png)

I hope that doesn't happen again :(

**Comment by mfranzs** [comment]
Thanks for reaching out. Do you ever see this happen where **more than one** card re-appears that you've always practiced? Or is it always one card?

**Comment by pormonto** [comment]
It can happen to any card where you press the "immediate" option then go back. So yes, this can happen to more than one card.
